pkgs/development/interpreters/python/sphinx-hook.nix

```diff
@@ -0,0 +1,6 @@
+{ makeSetupHook, python, findutils, sphinx }:
```

Comment:
there is a folder with hooks for Python packages ./pkgs/development/interpreters/python/hooks

Reply:
Fixed.